src/xdocs/config_naming.xml

```diff
@@ -1117,7 +1135,24 @@ for (int i = 1; i &lt; 10; i++) {}
   &lt;property name="format" value="^[a-z](_?[a-zA-Z0-9]+)*$"/&gt;
 &lt;/module&gt;
         </source>
-
+        <p>Code Example:</p>
+        <pre>
+class MyClass {
+   void MyMethod() {
```

Comment:
remove 1 space from this line and every line below include the final `}`.

Reply:
Thanks. Sorry for ignoring such a simple problem.